docs/content/reference/migration-0-9.md

```diff
@@ -3,4 +3,238 @@ title: Migration to 0.9
+   - `rr.log_extension_components(..., ext={'mydata': 1})` becomes `rr.log(... rr.AnyValues(mydata=1))`
+
+### `log_image`
+Replace with [Image](data_types/archetypes/image.md)
+
+Python docs: [Image.__init__](https://ref.rerun.io/docs/python/HEAD/common/images/#rerun.Image.__init__)
+
+Notes:
+ * `image` has become `data`
+ * `jpeg_quality` is now handled by calling `.compress(jpeg_quality=...)` on the image after constructing it.
+
+### `log_image_file`
+Replace with `ImageEncoded`
+
+Python docs: [ImageEncoded.__init__](https://ref.rerun.io/docs/python/HEAD/common/images/#rerun.ImageEncoded.__init__)
+
+Notes:
+ - `img_bytes` and `img_path`
+
+
+### `log_line_strip`, `log_line_strips_2d`, `log_line_strips_3d`, `log_line_segments`
+Replace with [LineStrips2D](data_types/archetypes/line_strips2d.md) or [LineStrips3D](data_types/archetypes/line_strips3d.md)
+
+Python docs: [LineStrips2D.__init__](https://ref.rerun.io/docs/python/HEAD/common/spatial_archetypes/#rerun.LineStrips2D.__init__), [LineStrips3D.__init__](https://ref.rerun.io/docs/python/HEAD/common/spatial_archetypes/#rerun.LineStrips3D.__init__)
+
+Notes:
+ - `log_line_segments` used to take an array of shape (2 * num_segments, 2 or 3) (where points were connected in
+even-odd pairs). Instead this is now handled by a batch of `LineStrips` all of length 2. Note that `LineStrips` now
+takes an array of shape (num_strips, num_points_per_strip, 2 or 3). You can use convert to the new format using the snippets:
+```
+line_strips2d=line_segments.reshape(-1, 2, 2)
+line_strips3d=line_segments.reshape(-1, 2, 3)
+```
+ - `positions` has become `strips`.
+ - `stroke_width` has become `radii`, which entails dividing by 2 as necessary.
+ - `identifiers` has become `instance_keys`.
+
+### `log_mesh`, `log_meshes`
+Replace with [Mesh3D](data_types/archetypes/mesh3d.md)
+
+Python docs: [Mesh3D.__init__](https://ref.rerun.io/docs/python/HEAD/common/spatial_archetypes/#rerun.Mesh3D.__init__)
+
+Notes:
+ - Meshes are no longer batch objects. Instead they are treated as a batch of vertices, as such there is no longer a
+   direct equivalent of `log_meshes`.
+ - `positions` has become `vertex_positions`.
+ - `normals` has become `vertex_normals`.
+ - `albedo_factor` has become `mesh_material`, and can be logged using `rr.Material(albedo_factor=...)`.
+ - `identifiers` has become `instance_keys`.
+
+### `log_mesh_file`
+Replace with [Asset3D](data_types/archetypes/asset3d.md)
+
+Python docs: [Asset3D.__init__](https://ref.rerun.io/docs/python/HEAD/common/spatial_archetypes/#rerun.Asset3D.__init__)
+
+Notes:
+ - `mesh_bytes` and `mesh_path` are both now jut `data`. Strings and paths will be opened as files, while
+   file-descriptors or bytes objects will be read.
+ - `mesh_format` is now `media_type`.
+ - `transform` can now take anything that is compatible with `rr.Transform3D` instead of a 3x4 matrix.
```

Comment:
We should write out how to turn a 3x4 matrix into a `Transform3D`

Reply:
Would be even easier if we implement: https://github.com/rerun-io/rerun/issues/3645